ocean_lib/ocean/ocean_assets.py

```diff
@@ -342,8 +343,17 @@ def create_bundled(
         name = metadata["name"]
         template_index = 2 if use_enterprise else 1
         data_nft_args = DataNFTArguments(name, name)
-        datatoken_args = DatatokenArguments(
-            f"{name}: DT1", files=files, template_index=template_index
+        datatoken_args = (
```

Comment:
I think you should assign this previously and simplify.
e.g. dt_args=DatatokenArguments(
                f"{name}: DT1", files=files, template_index=template_index
            )
            
            if template_index == 2:
            dt_args.cap=MAX_UINT256
```            
maybe the init part itself has the exception, but either way this needs to be cleaner

Reply:
Sure, thank you

Reply:
Added an assignment before initializing the DatatokenArguments object